als-suggestions/shared/src/main/scala/org/mulesoft/als/suggestions/plugins/aml/pathnavigation/PathNavigation.scala

```diff
@@ -49,25 +50,38 @@ private[pathnavigation] case class PathNavigation(
       case _         => Map.empty
     }
 
-  private def resolveRootNode(): Future[Option[YNode]] =
-    alsConfiguration.fetchContent(fileUri).map { c =>
-      val mime = c.mime.orElse(
-        alsConfiguration.platform
-          .extension(fileUri)
-          .flatMap(alsConfiguration.platform.mimeFromExtension)
-      )
-      mime.flatMap(pluginForMime) match {
-        case Some(plugin) =>
-          plugin
-            .parse(
-              c.stream,
-              mime.get,
-              ParserContext(config = ParseConfig(alsConfiguration.getAmfConfig, DefaultErrorHandler()))
-            ) match {
-            case SyamlParsedDocument(document, _) => Some(document.node)
-            case _                                => None
+  private def resolveRootNode(): Future[Option[YNode]] = {
+    try {
+      alsConfiguration
+        .fetchContent(fileUri)
+        .map { c =>
+          val mime = c.mime.orElse(
+            alsConfiguration.platform
+              .extension(fileUri)
+              .flatMap(alsConfiguration.platform.mimeFromExtension)
+          )
+          mime.flatMap(pluginForMime) match {
+            case Some(plugin) =>
+              plugin
+                .parse(
+                  c.stream,
+                  mime.get,
+                  ParserContext(config = ParseConfig(alsConfiguration.getAmfConfig, DefaultErrorHandler()))
+                ) match {
+                case SyamlParsedDocument(document, _) => Some(document.node)
+                case _                                => None
+              }
+            case _ => None
           }
-        case _ => None
-      }
+        }
+        .recoverWith { case _ =>
+          Future.successful(None)
+        }
+    } catch {
+      case _: Exception =>
+        val logger: Logger = PrintLnLogger
```

Comment:
no hagas esto, tenes que recibir el logger de mas arriba, nunca crear loggers nuevos asi
ademas, loggear el mensaje de excepcion (porque estas atajando cualquier exception, no solo las file not found)